src/rules/no-passthrough-getters.ts

```
@@ -0,0 +1,231 @@
+              messageId: 'noPassthroughGetter',
+            });
+          }
+        }
+      },
+    };
+
+    /**
+     * Check if the node is a simple property access from a constructor parameter
+     * like this.settings.property or this.settings['property'] or this.settings.nested.deep.property
+     */
+    function isConstructorParameterPropertyAccess(node: TSESTree.Expression): boolean {
```

ESLint

[error] Replace `node:·TSESTree.Expression` with `⏎······node:·TSESTree.Expression,⏎····`

(prettier/prettier)